xenserver/xenserver-pxe.erb

```diff
@@ -0,0 +1,5 @@
+default xenserver
+label xenserver
+kernel mboot.c32
+append boot/XenServer-6.5-x86_64-xen.gz dom0_max_vcpus=16 dom0_mem=752M com1=115200,8n1 console=com1,vga --- boot/XenServer-6.5-x86_64-vmlinuz xencons=hvc console=hvc0 console=tty0 answerfile=<%= foreman_url('provision')%> install --- boot/XenServer-6.5-x86_64-initrd.img
```

Comment:
Original files from ISO are named vmlinux.gz, vmlinuz and xen.gz. I renamed them on foreman host to right convention. User can copy them as it is and just use same name in template.